### PR TITLE
Add Quid Ika Token

### PR DESCRIPTION
**Pull request by ShailChoksi**
Adding Quid Ika Token (ERC20)

- website: https://quidverse.com/
- etherscan: https://etherscan.io/token/0x9d38f670d15c14716be1f109a4f453e966a2b6d4
- coingecko: https://www.coingecko.com/en/coins/quid-ika
- telegram: https://t.me/quidika

### CODE REVIEW

File: contract-map.json [hunk 1]
```diff
@@ -3805,6 +3805,13 @@
     "erc20": true,
     "symbol": "rETH",
 	"decimals": 18
+  },
+  {
+    "name": "Quid Ika",
+    "logo": "quidika.png",
```
Comment (MRabenda) [inline contract-map.json:3811]: png as weel as contract address is required

Comment (ShailChoksi) [reply contract-map.json:3811]: @MRabenda updated

**Comment by ShailChoksi** [comment]
@MRabenda bump?

**Comment by ShailChoksi** [comment]
Am I still missing something here?

File: contract-map.json (hunk 1 above)
Comment (MRabenda) [reply contract-map.json:3818]: Ups I made mistake SVG format of image is required to merge

**Comment by ShailChoksi** [comment]
@MRabenda updated

**Comment by MRabenda** [comment]
1. Please remove PNG from PR
2. Your SVG has ~ 800kb regular one should have < 10 - can you improve it?

**Comment by ShailChoksi** [comment]
> 1. Please remove PNG from PR
> 2. Your SVG has ~ 800kb regular one should have < 10 - can you improve it?

1. Done
2. I am not too familiar with SVGs so I am not able to get it much smaller before starting to lose details on the image. I cut it down to about 50% of the original.

**Comment by MRabenda** [comment]
From what I can see on Etherescan and Coinmarket cap Quid Ika has a different icon that this gray used here

**Comment by ShailChoksi** [comment]
> From what I can see on Etherescan and Coinmarket cap Quid Ika has a different icon that this gray used here

We have been rebranding. It takes time for CMC and Etherscan to update. Our Telegram, Discord and Dextools all have the new symbol/logo.

**Comment by ShailChoksi** [comment]
@MRabenda let me know if you need more info or need me to prove that this is the new logo for the token

**Comment by ShailChoksi** [comment]
Bump @MRabenda

**Comment by ShailChoksi** [comment]
@MRabenda bump?

**Review by MRabenda — APPROVED** [review]

**Comment by ShailChoksi** [comment]
How long does it take for this to show up in the app?

**Comment by MRabenda** [comment]
@ShailChoksi - QUID token has 18 or 9 decimals https://etherscan.io/address/0x9d38f670d15c14716be1f109a4f453e966a2b6d4#readContract ?

**Pull request merged**